drivers/ipm/ipm_mhu.c

```diff
@@ -195,18 +197,18 @@ DEVICE_AND_API_INIT(mhu_0,
 static void ipm_mhu_irq_config_func_0(struct device *d)
 {
 	ARG_UNUSED(d);
-	IRQ_CONNECT(DT_INST_0_ARM_MHU_IRQ_0,
-			DT_INST_0_ARM_MHU_IRQ_0,
+	IRQ_CONNECT(DT_INST_IRQN(0),
+			DT_INST_IRQN(0),
```

Comment:
While the conversion done here is correct, this does not look sane to me ...

Reply:
Agreed, seems like that should really be looking for `priority`.  Will fix it up.